packages/react-button/src/components/CompoundButton/CompoundButton.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Meta } from '@storybook/react';
+import { CompoundButton } from '../../CompoundButton';
+import descriptionMd from './CompoundButtonDescription.md';
+import bestPracticesMd from '../Button/ButtonBestPractices.md';
+
+export { Default } from './stories/CompoundButtonDefault.stories';
+export { Shape } from './stories/CompoundButtonShape.stories';
+export { Appearance } from './stories/CompoundButtonAppearance.stories';
+export { Icon } from './stories/CompoundButtonIcon.stories';
+export { Size } from './stories/CompoundButtonSize.stories';
+export { Disabled } from './stories/CompoundButtonDisabled.stories';
+export { WithLongText } from './stories/CompoundButtonWithLongText.stories';
+
+export default {
+  title: 'Components/CompoundButton',
+  component: CompoundButton,
+  parameters: {
+    docs: {
+      description: {
+        component: [descriptionMd, bestPracticesMd].join('\n'),
+      },
+    },
+  },
+  decorators: [
```

Comment:
I removed this decorator layout from Divider and found that it make it look better. Plus the example feels more encapsulated because all the styles are there.

Reply:
i started without the styles, but a little gap between each control i feel improves the look and would match what other docs do